File: Infy new/Models/Course.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Infy_new.Models
{
    public partial class Course
    {
        public Course()
        {
            UserCourseMaps = new HashSet<UserCourseMap>();
        }

        public int TrainerId { get; set; }
        public int CourseId { get; set; }
        public string Cname { get; set; }
        public int MinAge { get; set; }
        public int MaxAge { get; set; }
        public int Cdur { get; set; }
        public string PreReq { get; set; }
        public string Cdesc { get; set; }
        public bool Approval { get; set; }

        public virtual Trainer Trainer { get; set; }
        public virtual ICollection<UserCourseMap> UserCourseMaps { get; set; }
    }
}

File: Infy new/ClientApp/src/app/Interfaces/Course.ts
/* References https://infyspringboard.onwingspan.com/web/en/app/toc/lex_31364648044580110000/overview */
export interface ICourse {
  trainerId: number;
  courseId: number;
  cname: string;
  minAge: number;
  maxAge: number;
  cdur: number;
  preReq: string;
  cdesc: string;
  approval: boolean;
}
